pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -541,6 +598,13 @@ func parseAppInstanceConfig(config *zconfig.EdgeDevConfig,
 			appInstance.CloudInitUserData = &userData
 		}
 		appInstance.RemoteConsole = cfgApp.GetRemoteConsole()
+
+		// TBD:XXX Will enable once yetus issue is fixed,
+		// commented out for end-o-end testing with conttoller now
+		//if cipherBlock, appInstance.IsCipher := parseCipherBlock(getConfigCtx,
+		//	 cfgApp.GetCipherData()); cipherBlock != nil {
+		//	appInstance.CipherBlock = cipherBlock
+		//}
```

Comment:
Is this dead? You added the code a few lintes below.

Reply:
Will activate, once the yetus related issue is resolved.